src/chocolatey/infrastructure.app/configuration/EnvironmentSettings.cs

```diff
@@ -28,6 +28,7 @@ namespace chocolatey.infrastructure.app.configuration
     using logging;
     using nuget;
     using Environment = adapters.Environment;
+    using License = licensing.License;

```

Comment:
I don't think that this is needed, and can be removed.

Reply:
I was going to tell you Visual Studio added it (which is true), but I took out the `License.ValidateLicense()` calls that needed it, so you're absolutely right this is not needed. I've rebased away the commit that added this.